content/getting-started/using-docker.md

```diff
@@ -23,9 +29,11 @@ To compile `blinky1.hex` targeting an AVR microcontroller such as the Arduino:
 
     docker run --rm -v $(pwd):/src tinygo/tinygo tinygo build -o /src/blinky1.hex -size=short -target=arduino examples/blinky1
 
-For projects that have remote dependencies outside of the standard library and go code within your own project, you will need to map your entire GOPATH into the docker image in order for those dependencies to be found:
+For projects that have remote dependencies outside of the standard library and
```

Comment:
Maybe "with remote dependencies" ?

Reply:
Actually, maybe not... the "within" below makes that not sound quite right. :wink: